### PR TITLE
New flag system

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

File: implementation/v7/src/main/java/org/codemc/worldguardwrapper/implementation/v7/WorldGuardImplementation.java [hunk 1]
```diff
@@ -1,16 +1,14 @@
 package org.codemc.worldguardwrapper.implementation.v7;
 
+import com.sk89q.worldedit.BlockVector;
+import com.sk89q.worldedit.BlockVector2D;
```
Comment (EpicEricEE) [inline implementation/v7/src/main/java/org/codemc/worldguardwrapper/implementation/v7/WorldGuardImplementation.java]: As the sk89q repository has been fixed, the BlockVector and BlockVector2D don't exist anymore in the latest WorldEdit snapshot and have been replaced with BlockVector2 and BlockVector3. So, does this actually build?

File: library/src/main/java/org/codemc/worldguardwrapper/WorldGuardWrapper.java [hunk 2]
```diff
@@ -27,13 +27,20 @@ public static WorldGuardWrapper getInstance() {
     private Listener eventListener;
 
     private WorldGuardWrapper() {
+        String version;
         try {
             Class.forName("com.sk89q.worldguard.WorldGuard");
-            delegate = new org.codemc.worldguardwrapper.implementation.v7.WorldGuardImplementation();
```
Comment (EpicEricEE) [inline library/src/main/java/org/codemc/worldguardwrapper/WorldGuardWrapper.java]: Why don't we keep it like that?

File: implementation/v7/src/main/java/org/codemc/worldguardwrapper/implementation/v7/WorldGuardImplementation.java (hunk 1 above)
Comment (sgdc3) [reply implementation/v7/src/main/java/org/codemc/worldguardwrapper/implementation/v7/WorldGuardImplementation.java]: I'll try to force a dependency update in my dev environment :/

File: library/src/main/java/org/codemc/worldguardwrapper/WorldGuardWrapper.java (hunk 2 above)
Comment (sgdc3) [reply library/src/main/java/org/codemc/worldguardwrapper/WorldGuardWrapper.java]: it lets us handle the exceptions correctly

**Comment by sgdc3** [comment]
@EpicEricEE Ok, fixed the build ;)

**Pull request merged**